did-method-spec.md

```diff
@@ -0,0 +1,111 @@
+# TangleID DID Method Specification
+
+## About
+
+The TangleID DID method specification conforms to the requirements specified in the DID specification currently published by the W3C Credentials Community Group. For more information about DIDs and DID method specifications, please see the [DID Primer](https://github.com/WebOfTrustInfo/rwot5-boston/blob/master/topics-and-advance-readings/did-primer.md) and [DID Spec](https://w3c-ccg.github.io/did-spec/).
+
+## Abstraction
+
+IOTA is a public distributed ledger that utilizes an invention called the Tangle at its core, address scalability issues and having no transaction fee, that encourages adoption of the technology in the industry. TangleID is optimized for DIDs and DID Documents.
```

Comment:
Don't say "TangleID is optimized for DIDs and DID Documents." Instead, TangleID is intended to implement DIDs and DID Documents.

Reply:
Fixed